docs/enforce-docs-up-to-date.sh

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env bash
+
+set -e -o pipefail -u
+
+current_docs=$(cat git_machete/docs.py)
+generated_docs=$(python docs/generate_docs.py dont_save)
```

Comment:
I'd rather make the script print to stdout by default; saving can be done by a simple `> git_machete/docs.py`

Reply:
@PawelLipski Sure, it's done but i had to create separate bash file just for saving because tox doesn't support output redirections.

Reply:
Hmmm does it support piping to `tee`? Like `python ... | tee ...` (which is kinda equivalent to `python ... > ...`)

Reply:
@PawelLipski It does support `tee`, the issue was raised here https://github.com/tox-dev/tox/issues/73, the proposed workaround -> `commands =
    bash -c \'pylint --rcfile=.pylintrc implib2 |tee pylint.out\'`
unfortunately doesn't work but `bash -c "python docs/generate_py_docs.py |tee git_machete/docs.py"` works although it not only saves the the output but also prints it, is that acceptable?

Reply:
@PawelLipski Oh yeah, i got it, `bash -c "python docs/generate_py_docs.py > git_machete/docs.py"` redirects without printing, "ladies and gentlemen, we got him" :D

Reply:
Haha yeah, that's what I thought as well ;)